aws_lambda_powertools/utilities/idempotency/persistence/base.py

```diff
@@ -226,7 +225,6 @@ def _generate_hash(self, data: Any) -> str:
             Hashed representation of the provided data
 
         """
-        data = getattr(data, "raw_event", data)  # could be a data class depending on decorator order
```

Comment:
@heitorlessa - so doing it here is too late, so i am moved it up to the earliest possible point.

Reply:
GitHub UI didn't allow me to make a comment upward, tks a lot for moving it upward - I'll check it